lib.typ

```diff
@@ -3,6 +3,8 @@
   icon: none,
   column-number: 2,
   subtitle: none,
+  fill-color: "F2F2F2",
+  stroke-color: "21222C",
```

Comment:
The name probably should've been spelled as `colour` ;)

Reply:
Thanks for this merge request (may be too fast ;-)

As English is not my mother tongue I first thought I'd made a mistake but it seems that it is not that obvious : 
https://writingexplained.org/color-or-colour-difference.

So if you prefer colour instead of color I will just make a new PR and you'll have to choose to merge it or not ;-)

Reply:
Nah, I prefer to use `colour`, but in programming in most places I have to use the American version because that's what most people expect. Anyway, I didn't do my part well, because You missed the version update in a few spots, but that's alright.